apps/weather_status/src/App.vue

```diff
@@ -6,36 +6,56 @@
 <template>
 	<div id="weather-status-menu-item">
 		<NcActions class="weather-status-menu-item__subheader"
-			:default-icon="weatherIcon"
 			:aria-hidden="true"
 			:aria-label="currentWeatherMessage"
 			:menu-name="currentWeatherMessage">
+			<template #icon>
+				<NcLoadingIcon v-if="loading" />
+				<img v-else :src="weatherIconUrl" class="weather-image">
+			</template>
 			<NcActionText v-if="gotWeather"
-				:aria-hidden="true"
-				:icon="futureWeatherIcon">
+				:aria-hidden="true">
+				<template #icon>
+					<NcLoadingIcon v-if="loading" />
+					<div v-else class="weather-action-image-container">
+						<img :src="futureWeatherIconUrl" class="weather-image">
```

Comment:
Needs to be an empty alt tag to make it hidden in accessible tree:

```suggestion
						<img :src="futureWeatherIconUrl" alt="" class="weather-image">
```